tests/benchdnn/inputs/matmul/test_matmul_fp8

```diff
@@ -86,3 +86,15 @@
 8x2664:2664x256
 3x4128:4128x128
 
+## MXFP8
+--dt=f8_e4m3:f8_e4m3:f32,f8_e4m3:f8_e4m3:bf16
+--attr-scales=src:per_tensor:e8m0:1x32+wei:per_tensor:e8m0:32x1
+--batch=shapes_mx
```

Comment:
Since you added 4D shape into shapes_mx, you need to fix tags on line 83 now and then you could remove lines 98-100. Or tests will fail due to tag mismatch.